authapi/tests/test_organizations.py

```diff
@@ -544,6 +560,24 @@ def test_get_teams_for_organization(self):
 
         self.assertEqual(team['id'], team1.id)
 
+    def test_permission_teams_for_organization(self):
+        '''Any member of the organization should be able to see that
+        organization's teams.'''
+        org = SeedOrganization.objects.create()
+        url = reverse('seedorganization-teams-list', args=(org.pk,))
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
```

Comment:
Should we also check that users that are authenticated but not members of the organisation get `403`s?